app/api_topologies.go

```diff
@@ -31,6 +139,52 @@ type topologyStats struct {
 	FilteredNodes      int `json:"filtered_nodes"`
 }
 
+type byName []APITopologyDesc
```

Comment:
Let's move this up next to APITopologyDesc, to group it with the thing it sorts.